### ISSUE TITLE
Markdown file never loads in Marked 2

### ISSUE THREAD

**Issue by scottaw66** [jfieber/Marked2.novaextension#3]
Marked 2.6.13 (1027)
Nova 8.4
Marked 2 extension 1.1.0

Using the Editor > View in Marked 2 option with a markdown file loaded in the editor does nothing. Nothing opens in marked 2 whatsoever. Makes no difference if I have Marked 2 already running or not

https://user-images.githubusercontent.com/7348442/156283898-dcfca320-7e3f-4d5a-8cdb-5c6d4cce8821.MP4

.

**Comment by jfieber** [comment]
@scottaw66 can you open the Extension Console (_Show Extension Console_ at the bottom of the _Extensions_ menu) and see if anything with a Source of "Marked 2" is logged there?

**Comment by scottaw66** [comment]
<img width="370" alt="Screen Shot 2022-03-05 at 1 57 31 PM" src="https://user-images.githubusercontent.com/7348442/156901154-35eab37d-9cbb-43a9-9741-060d61785f95.png">

**Comment by scottaw66** [comment]
Sorry, never mind... I figured the extension would look in Applications folder for Marked 2 but I had to set the path explicitly in preferences.

Is it possible to have the extension check there first?

**Comment by jfieber** [comment]
@scottaw66 It _should_ find the application regardless of where it is installed. It should work even if you rename the application, because it looks for the bundle identifier, not the app name. There is some complexity around the default behavior because there are three distinct distributions of Marked: Setapp bundle, direct download, and App Store.  It first looks for the newest version, then if there are multiple distributions of the same version, it prefers Setapp over direct download, and direct download over App Store.

Anyway, the error logged is…unhelpful to be sure. Would you be able to run this command in a Terminal window and let me know if it displays the path to your Marked 2 application, or some other error?  Thanks.

```
mdfind kMDItemCFBundleIdentifier = 'com.brettterpstra.marked*'
```

**Comment by scottaw66** [comment]
it returns /Applications/Marked 2.app

**Comment by jfieber** [comment]
A silly error, found and fixed in v1.1.1 just pushed to the extension library.
